### ISSUE TITLE
Certificate for lists.buildbot.net expired

### ISSUE THREAD

**Issue by seankelly** [buildbot/buildbot-infra#184]
From IRC:

```
09:42:42 < Mosu> Hey. https://lists.buildbot.net sends a certificate whose validity 
                 has expired nine days ago. Due to HSTS my browser doesn't allow me to 
                 visit the page either. Just as a heads-up.
```

The certificate is from Let's Encrypt. @tardyp this should auto-update, right? buildbot.net is working fine, so it's only lists.

**Comment by seankelly** [comment]
```
# /root/.acme.sh/acme.sh --cron --home /root/.acme.sh
[Wed Jul 12 15:08:23 UTC 2017] ===Starting cron===
[Wed Jul 12 15:08:23 UTC 2017] Renew: 'lists.buildbot.net'
[Wed Jul 12 15:08:23 UTC 2017] Skip, Next renewal time is: Thu Aug  3 00:52:12 UTC 2017
[Wed Jul 12 15:08:23 UTC 2017] Add '--force' to force to renew.
[Wed Jul 12 15:08:23 UTC 2017] Skipped lists.buildbot.net
[Wed Jul 12 15:08:23 UTC 2017] ===End cron===
```

**Comment by seankelly** [comment]
There's a command in the nginx role to instruct acme.sh how to install the certificate. Anything not using nginx will thus not get the updated certificates.

I checked root's history on lists and someone manually ran `/root/.acme.sh/acme.sh --install-cert ...` but didn't include the `--reloadcmd` option. I added that and the certificate reloaded. I suspect any other Apache-run sites using TLS will have the same issue.
